crates/forge/tests/e2e/running.rs

```diff
@@ -782,6 +782,7 @@ fn incompatible_snforge_std_version_warning() {
         .parse::<DocumentMut>()
         .unwrap();
     scarb_toml["dev-dependencies"]["snforge_std"] = value("0.34.1");
+    scarb_toml["dev-dependencies"]["snforge_scarb_plugin"] = value("0.34.1");
```

Comment:
After we add proper tests for plugin reexport we might want to revisit this. And use version that supports reexports here.

Please link issue #3069 here